training_laravel/app/Http/Controllers/UsersController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+use Illuminate\Http\Request;
+
+class UsersController extends Controller {
+
+	public function index() {
+		$users = User::all();
+
+		return view('users.index', compact('users'));
```

Comment:
@sshun01 Can you please change indentation here to 4 spaces instead of tabs? We will follow standard so everything is uniform.